### PR TITLE
add(epp): google workspace api proc - deprecate old google gsuite proc

### PR DESCRIPTION
**Pull request by garnier-quentin**
## Description
-

## Target serie

- [X] 20.04.x
- [X] 20.10.x
- [X] 21.04.x
- [X] 21.10.x (master)

### CODE REVIEW

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+---
+
+## Overview
+
+Google Workspace, is a collection of cloud computing, productivity and collaboration tools, software and products developed and marketed by Google.
+
+The Centreon Plugin Pack aims to collect the real-time status and availability of the services by requesting the
+dedicated Status API available on Google servers.
+
+## Plugin-Pack assets
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:13]: Plugin Pack

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+## Monitored metrics 
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Services-->
+
+| Metric name                     | Description                            |
+| :------------------------------ | :------------------------------------- |
+| google.workspace.services.count | Number of services currently monitored |
+| status                          | status of the service                  |
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:43]: uppercase ;)

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+<!--Online IMP Licence & IT-100 Editions-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor *Google Workspace* services:
+
+```bash
+yum install centreon-plugin-Applications-Google-Workspace-Api
+```
+
+2. On the Centreon Web interface, install the *Google Workspace* Centreon Plugin Pack on the "Configuration > Plugin Packs > Manager" page
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:66]: s/Plugin Pack/Pack/

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+<!--Offline IMP License-->
+
+1. Install the Centreon Plugin package on every Centreon poller expected to monitor *Google Workspace* services:
+
+```bash
+yum install centreon-plugin-Applications-Google-Workspace-Api
+```
+
+2. Install the Centreon Plugin Pack RPM on the Centreon Central server:
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:76]: s/Plugin Pack/Pack/

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+```bash
+yum install centreon-pack-applications-google-workspace-api
+```
+
+3. On the Centreon Web interface, install the *Google Workspace* Centreon Plugin Pack on the "Configuration > Plugin Packs > Manager" page
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:82]: s/Plugin Pack/Pack/

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Configuration
+
+* Log into Centreon and add a new Host through "Configuration > Hosts".
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:88]: Use bold for menus `**Configuration > Hosts**`

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md (hunk 3 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:66]: Use bold for menus `**Configuration > Hosts**`

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md (hunk 5 above)
Comment (Sims24) [reply en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:82]: Use bold for menus `**Configuration > Hosts**`

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 7]
```diff
@@ -0,0 +1,160 @@
...
+| X         | GOOGLEWORKSPACESTATUSHOSTNAME | IP or name of the Status server (Default: 'www.google.com')                                 |
+| X         | GOOGLEWORKSPACESTATUSPORT     | Port used to reach the Google server (Default: '443')                                       |
+| X         | GOOGLEWORKSPACESTATUSPROTO    | Protocol used to reach the Google server (Default: 'https')                                 |
+|           | PROXYURL                      | Configure a proxy URL to use if needed                                                      |
+|           | GOOGLEWORKSPACEEXTRAOPTIONS   | Any extra option you may want to add to every command\_line (eg. a --verbose flag)          |
+|           | DUMMYSTATUS                   | Host state. Default is OK, do not modify it unless you know what you are doing              |
+|           | DUMMYOUTPUT                   | Host check output. Default is 'This is a dummy check'. Customize it with your own if needed |
+
+> This template will deploy one "Global" Service that will monitor all of the services.
+> Use the **Service Discovery** feature if you wish to get one Service per google workspace service.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:106]: Is it still true?

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 8]
```diff
@@ -0,0 +1,160 @@
...
+    --plugin='apps::google::workspace::plugin' \
+    --mode=services \
+    --help
+```
+
+### Why do I get the following message: ```UNKNOWN: 500 Can't connect to www.google.com.443 |```
+
+This error message means that the Centreon Plugin couldn't successfully connect to the Google API.
+Check that no third party device (such as a firewall) is blocking the request.
+A proxy connection may also be necessary to connect to the API.
```
Comment (Sims24) [inline en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:159]: Link to dedicated topic. 

Missing ## Troubleshooting header

File: fr/integrations/plugin-packs/procedures/applications-google-workspace-api.md [hunk 9]
```diff
@@ -0,0 +1,161 @@
+---
+id: applications-google-workspace-api
+title: Google Workspace
+---
+
```
Comment (Sims24) [inline fr/integrations/plugin-packs/procedures/applications-google-workspace-api.md:5]: Apply EN recommendation unless troubleshooting one.

**Review by Sims24 — CHANGES_REQUESTED** [review]

File: en/integrations/plugin-packs/procedures/applications-google-workspace-api.md (hunk 7 above)
Comment (garnier-quentin) [reply en/integrations/plugin-packs/procedures/applications-google-workspace-api.md:104]: yes still true

**Review by Sims24 — APPROVED** [review]

**Pull request merged**